system/ee/ExpressionEngine/View/_shared/footer.php

```diff
@@ -1,11 +1,10 @@
-                <div id="output_profiler"></div>
             </div>
         </section>
     </div>
 
     <?php if (IS_PRO) : ?>
     <div class="dropdown app-pro-validation" data-dropdown="app-pro-validation-dropdown">
-      <h5>ExpressionEngine Pro</h5>
+      <h2>ExpressionEngine Pro</h2>
```

Comment:
Is there a specific reason for this to be `h2` and not something less standing out such as `h4`?

Reply:
I changed `h5 -> h2` to prevent heading levels from being skipped
from doc: `Headings provide document structure and facilitate keyboard navigation by users of assistive technology. These users may be confused or experience difficulty navigating when heading levels are skipped.`